src/apps/relay/http_server.c

```diff
@@ -172,6 +172,7 @@ static struct http_request *parse_http_request_1(struct http_request *ret, char
           if (evhttp_parse_query_str(query, kv) < 0) {
             free(ret);
             ret = NULL;
+            if (kv) free(kv); //kv no longer assigned on this path
```

Comment:
you'll need braces around this if-check to pass the linter.

Reply:
Done!